test/OpenSpaceToolkit/Astrodynamics/Dynamics/Tabulated.test.cpp

```diff
@@ -251,10 +276,7 @@ TEST_F(OpenSpaceToolkit_Astrodynamics_Dynamics_Tabulated, ComputeContribution)
             loadData("/app/test/OpenSpaceToolkit/Astrodynamics/Dynamics/Tabulated_Earth_Gravity.csv");
 
         const Tabulated tabulated = {
-            instants,
-            contributionProfile,
-            defaultWriteCoordinatesSubsets_,
-            defaultFrameSPtr_,
+            instants, contributionProfile, defaultWriteCoordinatesSubsets_, defaultFrameSPtr_, defaultInterpolationType_
```

Comment:
```suggestion
            instants, contributionProfile, defaultWriteCoordinatesSubsets_, defaultFrameSPtr_, defaultInterpolationType_,
```
And fix the formatting.